The shlibs files for hppa64 and cross binutils are unuseable, because there is
no -dev package containing the unversioned .so symbolic link. Thus stop
installing the (useless) shlibs files.

Doing so avoids splitting binutils into further packages. Only the shlibs file
for binutils itself is left and requires introducing a libbinutils package in
the next step.

Index: debian/binutils-cross.shlibs.in
===================================================================
--- debian.orig/binutils-cross.shlibs.in
+++ /dev/null
@@ -1,2 +0,0 @@
-libbfd @VER@-@ARCH@@DATE_EXT@ @PKG@ (>= @DEB_UVER@), @PKG@ (<< @DEB_NVER@)
-libopcodes @VER@-@ARCH@@DATE_EXT@ @PKG@ (>= @DEB_UVER@), @PKG@ (<< @DEB_NVER@)
Index: debian/binutils-hppa64-linux-gnu.shlibs.in
===================================================================
--- debian.orig/binutils-hppa64-linux-gnu.shlibs.in
+++ /dev/null
@@ -1,2 +0,0 @@
-libbfd @VER@-hppa64@DATE_EXT@ binutils-hppa64-linux-gnu
-libopcodes @VER@-hppa64@DATE_EXT@ binutils-hppa64-linux-gnu
Index: debian/rules
===================================================================
--- debian.orig/rules
+++ debian/rules
@@ -1396,14 +1396,6 @@
 	fi; \
 	date_ext=$(DATE_EXT); \
 	[ "$(is_rc)" = yes ] && date_ext=; \
-	sed -e 's/@VER@/$(VERSION)/g' \
-		-e 's/@DEB_VER@/$(DEB_VERSION)/g' \
-		-e 's/@DEB_UVER@/$(DEB_UPSTREAM)/g' \
-		-e "s/@DEB_NVER@/$$nver/g" \
-		-e "s/@DATE_EXT@/$$date_ext/g" \
-		-e 's/@ARCH@/$*/g' \
-		-e "s/@PKG@/$(P_CROSS)/g" \
-	    debian/binutils-cross.shlibs.in > $(D_CROSS)/DEBIAN/shlibs
 	$(install_file) debian/binutils.triggers $(D_CROSS)/DEBIAN/triggers
 
 	rm -f debian/substvars
@@ -1517,7 +1509,6 @@
 ifeq ($(with_hppa64),yes)
 	$(install_dir) $(d_hppa64)/DEBIAN
 	$(install_file) debian/binutils.triggers $(d_hppa64)/DEBIAN/triggers
-	$(install_file) debian/$(p_hppa64).shlibs $(d_hppa64)/DEBIAN/shlibs
 endif
 
 	: # install docs
